### ISSUE TITLE
Macro does not run for player user in v11 any more 

### ISSUE THREAD

**Issue by KristjanLaane** [mclemente/fvtt-advanced-macros#71]
Macro is attached as a zip. 
[fvtt-Macro-itemmeasuredtemplate-UUq7vzX3grCaRhCB.zip](https://github.com/mclemente/fvtt-advanced-macros/files/11886569/fvtt-Macro-itemmeasuredtemplate-UUq7vzX3grCaRhCB.zip)

I call it from my add-on like so:
```
        const macro = game.macros?.find(m => m.name === "ItemMeasuredTemplate")
        if (!macro) return
        macro.execute({item: this.item5e})
```
When the user for macro is set to None it works fine. 
But when set macro to run for a player user, then nothing happens. No error messages, no nothing. 
(I run the add-on code to execute the macro from my Gamemaster user while the player user is logged into the server)

Using Foundry 11.302

**Comment by KristjanLaane** [comment]
Firstly, am I making a most basic mistake: does the player user have to have the macro also?

Secondly, it seems to be nothing to do with passing arguments, the following macro does not run either:
```
console.log("blah blah blah")
ui.notifications?.info("yes yes yes")
```

**Comment by mclemente** [comment]
Should be fixed now on v2.0.2.

**Comment by KristjanLaane** [comment]
Yes indeed! I updated to v2.0.2 and at first it started showing don’t have permission to run the macro warning on the player user side. Then I gave Limited permission to the user and works perfectly. Thank you!
